libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -375,6 +372,33 @@ public void track(AnalyticsTracker.Stat stat, Map<String, ?> properties) {
             case THEMES_DETAILS_ACCESSED:
                 eventName = "themes_details_accessed";
                 break;
+            case SETTINGS_LANGUAGE_SELECTION_FORCED:
+                eventName = "settings_language_selection_forced";
```

Comment:
If we change the KEY to `SITE_SETTINGS_LANGUAGE_SELECTION_FORCED` we need to change the value here.